native/index.d.ts

```diff
@@ -0,0 +1,68 @@
+// <reference path="../react/react-native.d.ts" />
+
+declare module "styled-components/native" {
```

Comment:
I removed the declare module blocks, I haven't moved yet the library outside the "typings" folder since the lib folder is created at build time.

Can you have a look at this comment? https://github.com/styled-components/styled-components/pull/152#discussion_r89686267

Should be the last step :) 
Thank you very much for the reviews, really appreciated!